### DISCUSSION TITLE
Manual/class-QualitySettings

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#294] [Notes]
# Manual/class-QualitySettings

https://docs.unity3d.com/Manual/class-QualitySettings.html

**Comment by joonjoonjoon** [comment]
I am convinced that Maximum LOD Level should be renamed to Minimum LOD level.  
To the best of my understanding, if Maximum LOD level is set to 1, it will exclude LOD0 (highest quality) from builds and default to LOD1 (lower quality).
